packages/zoe/test/unitTests/contractSupport/test-bondingCurves.js

```diff
@@ -144,3 +147,43 @@ test('getCurrentPrice ok 6', t => {
     t.assert(false, e);
   }
 });
+
+test('calculate extent to mint - positive supply', t => {
+  const res = calcLiqExtentToMint({
+    liqTokenSupply: 20,
+    inputExtent: 30,
+    inputReserve: 5,
+  });
+  t.equals(res, (20 * 30) / 5, 'When supply is present, floor(x*y/z)');
+  t.end();
+});
+
+test('calculate extent to mint - mispelled key', t => {
+  const res = calcLiqExtentToMint({
+    liquidityTokenSupply: 20,
```

Comment:
If keyword is misspelled, I think we should throw

Reply:
done